modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/model/listener/SystemObjectDefinitionManagerModelListener.java

```diff
@@ -374,6 +424,12 @@ private void _validateSystemObject(T originalModel, T model)
 	private static final Log _log = LogFactoryUtil.getLog(
 		SystemObjectDefinitionManagerModelListener.class);
 
+	private static final ThreadLocal<Boolean> _skipValidateReadOnly =
```

Comment:
Can we reuse `_skipValidateReadOnly` property from `ObjectEntryThreadLocal` class?